mod/item.php

```diff
@@ -346,14 +346,14 @@ function item_post(App $a) {
 	if ($parent) {
 		if ($thr_parent_contact['network'] == NETWORK_OSTATUS) {
 			$contact = '@[url=' . $thr_parent_contact['url'] . ']' . $thr_parent_contact['nick'] . '[/url]';
-			if (!in_array($contact, $tags)) {
+			if (!stripos(implode($tags), '[url=' . $thr_parent_contact['url'] . ']')) {
```

Comment:
Aren’t you supposed to provide a glue parameter?

Reply:
Doesn't seem so: http://php.net/manual/en/function.implode.php

Reply:
Interesting, I never used it like this.